subgraphs/alpaca-finance-lending/README.md

```diff
@@ -0,0 +1,66 @@
+# Alpaca Finance Lending Protocol Subgraph
+
+## Calculation Methodology v1.0.0
+
+Alpaca Finance is a leading lending protocol allowing leveraged yield farming on BNB Chain. It helps lenders earn safe and stable yields, and offers borrowers undercollateralized loans for leveraged yield farming positions, vastly multiplying their farming principals and resulting profits.‌
+
+### Total Value Locked (TVL) USD
+
+Sum across all Reserves:
```

Comment:
Should we also include the deposits in active positions and deposits in their automated vaults.

When borrowing from alpaca finance you have to put down a certain amount which is not deposited in the lending pools. Should that add to the TVL? Or not because it is just passed through to Pancake/Biswap/etc?

Also, alpaca tokens locked in their governance contract?

Reply:
@ishraq8 thoughts on TVL here?

Reply:
@jaimehgb 
1) Governance tokens that are locked or staked we do not include in TVL (similar to other tokens staked in their protocol). 
2) The token/stablecoin amount used for yield farming should be included in tvl, it works just like a yield aggregator added to their lending component based on their docs. Note: this should not be the interest bearing token that is produced from lending (receipt token) that is used in farming. 

Does that make sense? Simple answer: Include all non Alpaca tokens into TVL and exclude tokens produced by the protocol itself.

Reply:
@ishraq8 @jaimehgb @this-username-is-taken 

Totally agree with the governance token is not included in TVL. 

For the second one, as this subgraph focus on the alpaca finance lending protocol and doesn't include the yield farming protocol, I think the token used for the yield farming should not be included in the TVL for the lending protocol, as the token is not deposited in the lending protocol. As you can see from https://app.alpacafinance.org/lend and https://app.alpacafinance.org/farm, alpaca have different pools for lending and yield farming and this subgraph only take the lending pool into consideration. The total supply computed with this method from the subgraph also are in line with the data shown from their website at https://app.alpacafinance.org/lend

Reply:
@corerouter Agreed with second point, their farms are auto compounders/yield aggregator and would require a second subgraph for the farms using yield aggregator. Thanks for the clarification

@jaimehgb In that case the TVL is fine as is.